barretenberg/acir_tests/sol-test/src/index.js

```diff
@@ -7,10 +7,13 @@ import solc from "solc";
 // Size excluding number of public inputs
 const NUMBER_OF_FIELDS_IN_PLONK_PROOF = 93;
 const NUMBER_OF_FIELDS_IN_HONK_PROOF = 443;
+const NUMBER_OF_FIELDS_IN_HONK_ZK_PROOF = 494;
```

Comment:
This number is confusing me, how is it one larger when there are multiple new commitments 
- 4 new libraPolyEvals
- 3 new libra commitments
- 1 new libra sum
- 1 new libra evaluation